src/libostree/ostree-repo-pull.c

```diff
@@ -3570,6 +3588,9 @@ ostree_repo_pull_with_options (OstreeRepo *self, const char *remote_name_or_base
                               &pull_data->timestamp_check_from_rev);
       (void)g_variant_lookup (options, "max-metadata-size", "t", &pull_data->max_metadata_size);
       (void)g_variant_lookup (options, "append-user-agent", "s", &pull_data->append_user_agent);
+      (void)g_variant_lookup (options, "low-speed-limit-bytes", "u", &pull_data->low_speed_limit);
```

Comment:
Though for this one let's also do what we do above for
```
  if (!opt_n_network_retries_set)
    pull_data->n_network_retries = DEFAULT_N_NETWORK_RETRIES;
```
or so to ensure the defaults are consistently inherited?